libs/components/tabs/src/lib/modules/tabs/fixtures/tabset-adapter.service.mock.ts

```diff
@@ -1,19 +1,40 @@
-import { Injectable } from '@angular/core';
+import { Injectable, OnDestroy } from '@angular/core';
+
+import { BehaviorSubject, Observable } from 'rxjs';
 
 import { SkyTabsetAdapterService } from '../tabset-adapter.service';
 
 @Injectable()
-export class MockTabsetAdapterService extends SkyTabsetAdapterService {
+export class MockTabsetAdapterService
```

Comment:
The changes to this file are because the previous functionality relied on `_overflowChange` and `currentOverflow` being accessible by the extending class. However, with the hash private variables that was no longer possible. I modified this class so that we can still use its basic functionality while still mocking the overflow changes.